src/app/complex/complex-results/complex-filter/complex-filter.component.ts

```diff
@@ -43,7 +43,10 @@ export class ComplexFilterComponent implements OnInit {
 
   onResetAllFilters = output<boolean>();
 
-  confidenceGap = input<number>(2);
+  /**
```

Comment:
Yep, basically the slider has a minimum length that I couldn't overwrite, at least easily, so the minimum space between stars in order for the star pattern to take as much space as the slider is 5. If you go bellow, the area gets ddimisnished, but not the slider, leading it to be over the next area, and not alligend with the stars.
5 is pixel size

Reply:
Then the comment should be something like
```
Slider length cannot be overwritten. In order for the stars to take the same space as the slider, the minimum space between stars must be 5, otherwise they don't align.
```